### PR TITLE
Many tables are missing

### PR DESCRIPTION
**Pull request by indanielo**
These tables are missing:
L1_INFO_TREE_UPDATES_BY_GER
L1_BATCH_DATA
L1_INFO_TREE_HIGHEST_BLOCK
REUSED_L1_INFO_TREE_INDEX
LATEST_USED_GER
BATCH_BLOCKS
SMT_DEPTHS
L1_INFO_LEAVES
L1_INFO_ROOTS
INVALID_BATCHES
BATCH_PARTIALLY_PROCESSED
LOCAL_EXIT_ROOTS
ROllUP_TYPES_FORKS
FORK_HISTORY

However 3 of them do not allow to turn on RPC daemon: BATCH_PARTIALLY_PROCESSED, ROllUP_TYPES_FORKS and FORK_HISTORY

### CODE REVIEW

File: kv/tables.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ const (
 	//key - contract code hash
 	//value - contract TEVM code
 	ContractTEVMCode = "TEVMCode"
-)
+)
```
Comment (revitteth) [inline kv/tables.go:182]: Unnecessary whitespace change.

**Comment by indanielo** [comment]
Can you please take a look at this issue? Basically, what's happening is that the RPCs are not working as they should because some database buckets don't exist when the datadir is passed, and the system is using the local database. If the connection is remote via protobuf, then there's no issue because the database is properly initialized. I understand this is a very serious problem that affects anyone working with RPC on the same machine as the node, since the speed is significantly faster. Thank you.

**Comment by revitteth** [comment]
Addressed in a subsequent PR
